tyk-pro/scripts/bootstrap_k8s.sh

```diff
@@ -1,196 +1,28 @@
 #!/bin/bash
 
-if [ -z "${2}" ]; then 
-    ADMIN_TOKEN='12345'
-else 
-    ADMIN_TOKEN=${2}
-fi
+set -e
 
-if [ -z "${3}" ]; then 
-   echo "namespace is required"
-   exit
-else 
-    NS=${3}
-fi
+apk --no-cache add curl
```

Comment:
Fine for now, but we should later create an image with everything needed in place already.

Reply:
Image I'm using is `bash:5.1` and it only got bash on it. It's only 5MB, so I believe we're good here.